client/my-sites/email/email-management/gsuite-users-card/index.jsx

```diff
@@ -23,19 +23,17 @@ import Notice from 'components/notice';
 import PendingGSuiteTosNotice from 'my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice';
 import SectionHeader from 'components/section-header';
 import { withLocalizedMoment } from 'components/localized-moment';
+import { hasTitanMailWithUs } from 'calypso/lib/titan/has-titan-mail-with-us';
+import TitanControlPanelLoginCard from 'calypso/my-sites/email/email-management/titan-control-panel-login-card';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
 class GSuiteUsersCard extends React.Component {
-	getDomainsAsList() {
-		return this.props.selectedDomainName ? [ getSelectedDomain( this.props ) ] : this.props.domains;
-	}
-
 	canAddUsers( domainName ) {
-		return this.getDomainsAsList().some(
+		this.props.domainsAsList.some(
```

Comment:
I was wondering why the `Add New User` button would no longer be displayed for G Suite accounts. I took me some time to find the culprit: we are missing a `return` statement here.